ssas/service/admin/api.go

```diff
@@ -499,6 +499,48 @@ func getSystemIPs(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+/*
+	swagger:route DELETE /system/{system_id}/ip/{ip_id} system deleteSystemIP
+
+	Delete IP
+
+	Soft-deletes the IP of the associated system. Returns the deleted IP.
+
+	Produces:
+	- application/json
+
+	Security:
+		basic_auth:
+
+	Responses:
+		200: okResponse
+		400: badRequestResponse
+		500: serverErrorResponse
+		404: notFoundResponse
+*/
+func deleteSystemIP(w http.ResponseWriter, r *http.Request) {
+	systemID := chi.URLParam(r, "systemID")
+	system, err := ssas.GetSystemByID(systemID)
+	if err != nil {
+		jsonError(w, http.StatusNotFound, "Invalid system ID")
+		return
+	}
+
+	ipID := chi.URLParam(r, "id")
+
+	trackingID := uuid.NewRandom().String()
+	ssas.OperationCalled(ssas.Event{Op: "DeleteSystemIP", TrackingID: trackingID, Help: "calling from admin.deleteSystemIP()"})
+
+	err = system.DeleteIP(ipID, trackingID)
+	if err != nil {
+		//ssas.OperationFailed
```

Comment:
On line 532, if you create a reference to the `ssas.Event` you can reuse it here. Then we can uncomment. 😄

Reply:
Thanks David! I updated with a reference to the event and also made tracking ID generation more consistent across the file. 👍🏻